docs/_entities/account_migration.md

```diff
@@ -0,0 +1,57 @@
+---
+title: AccountMigration
+---
+
+This entity is sent when a person changes their diaspora* ID (e.g. when a user migration from one to another pod happens).
+
+## Properties
+
+| Property    | Type                         | Description                                                                          |
+| ----------- | ---------------------------- | ------------------------------------------------------------------------------------ |
+| `author`    | [diaspora\* ID][diaspora-id] | The diaspora\* ID of the closed account.                                             |
+| `person`    | [Profile][profile]           | New profile of a person                                                              |
+| `signature` | [Signature][signature]       | Signature that validates original and target diaspora* IDs with the new key of person |
+
+### Signature
+
+The string that is being used to generate the signature or verify against it follows the pattern:
+
+~~~
+AccountMigration:#{old_diaspora_id};#{new_diaspora_id}
```

Comment:
Maybe it's okay to describe this in words. We have an example there, so we having a pattern is not necessary